api_app/script_analyzers/observable_analyzers/firehol_iplist.py

```diff
@@ -0,0 +1,70 @@
+import logging
+import re
+import traceback
+import requests
+import ipaddress
+
+from api_app.exceptions import AnalyzerRunException
+from api_app.script_analyzers import classes
+
+logger = logging.getLogger(__name__)
+
+
+class FireHol_IPList(classes.ObservableAnalyzer):
+    def set_config(self, additional_config_params):
+        self.list_names = additional_config_params.get("list_names", [])
+
+    def run(self):
+        result = {}
+
+        for list_name in self.list_names:
+            result[list_name] = False
+            db = self.download_iplist(list_name)
```

Comment:
I guess you did not check the stratosphere analyzer as I mentioned....we cannot scrape the site each time an analysis is done...

Reply:
My bad, I will change accordingly.